Tasks/AzureKeyVault/operations/KeyVault.ts

```diff
@@ -41,14 +45,21 @@ export class KeyVault {
 
     private taskParameters: keyVaultTaskParameters.KeyVaultTaskParameters;
     private keyVaultClient: armKeyVault.KeyVaultClient;
+    private provisionKeyVaultSecretsScript: string;
 
     constructor(taskParameters: keyVaultTaskParameters.KeyVaultTaskParameters) {
         this.taskParameters = taskParameters;
+
         this.keyVaultClient = new armKeyVault.KeyVaultClient(
             this.taskParameters.vaultCredentials, 
             this.taskParameters.subscriptionId,
             this.taskParameters.keyVaultName,
             this.taskParameters.keyVaultUrl);
+
+        this.provisionKeyVaultSecretsScript = util.format("$ErrorActionPreference=\"Stop\";Login-AzureRmAccount -SubscriptionId %s;$spn=(Get-AzureRmADServicePrincipal -SPN %s);$spnObjectId=$spn.Id;Set-AzureRmKeyVaultAccessPolicy -VaultName %s -ObjectId $spnObjectId -PermissionsToSecrets get,list;",
```

Comment:
Nitpick: Would it be better to have this script in a much more readable form with new lines since we expect customer to read this script before running it.